src/main/features/claimant-response/routes/payment-plan.ts

```diff
@@ -92,8 +93,22 @@ export class PaymentPlanPage extends AbstractPaymentPlanPage<DraftClaimantRespon
     if (decisionType === DecisionType.DEFENDANT) {
 
       if (claimResponse.paymentIntention.paymentOption === PaymentOption.INSTALMENTS) {
-        courtOfferedPaymentIntention.repaymentPlan = claimResponse.paymentIntention.repaymentPlan
-        courtOfferedPaymentIntention.paymentOption = PaymentOption.INSTALMENTS
+        const paymentPlanWhenNoDisposableIncome: PaymentPlan = defendantPaymentPlan.convertTo(
+          defendantPaymentPlan.frequency,
+          claimantEnteredPaymentPlan.startDate)
+
+        if (paymentPlanFromDefendantFinancialStatement.instalmentAmount === 0) {
```

Comment:
if anyone can think of any better way to check this condition, please let me know. 

we have this to generate payment plan when disposable income is <1. 

`const instalmentAmount: number = Math.min(draft.courtDetermination.disposableIncome / Frequency.WEEKLY.monthlyRatio, claim.totalAmountTillToday)
    if (instalmentAmount < 1) {
      return PaymentPlanHelper.createPaymentPlanFromStartDate(MomentFactory.maxDate())
    }`

`private static createPaymentPlanFromStartDate (firstPaymentDate: Moment): PaymentPlan {
    return PaymentPlan.create(0, 0, Frequency.WEEKLY, firstPaymentDate)
  }`